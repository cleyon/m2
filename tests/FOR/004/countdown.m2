@for I 5 1 -1
I is @I@
@next I
