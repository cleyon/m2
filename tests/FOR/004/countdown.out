I is 5
I is 4
I is 3
I is 2
I is 1
